omod/pom.xml

```diff
@@ -89,6 +89,12 @@
 			<scope>test</scope>
 		</dependency>
 
+		<dependency>
+			<groupId>org.openmrs.module</groupId>
+			<artifactId>legacyui-omod</artifactId>
+			<type>jar</type>
```

Comment:
The `<type/>` tag is not necessary, however you should add `<scope>provided</scope>` here. Can you check at runtime?